pkg/graph/schema.resolvers.go

```diff
@@ -1875,12 +1875,37 @@ func (r *mutationResolver) SendReportAProblemEmail(ctx context.Context, input mo
 
 // CreateTRBRequest is the resolver for the createTRBRequest field.
 func (r *mutationResolver) CreateTRBRequest(ctx context.Context, requestType models.TRBRequestType) (*models.TRBRequest, error) {
-	return resolvers.TRBRequestCreate(ctx, requestType, r.store)
+	return resolvers.CreateTRBRequest(ctx, requestType, r.store)
 }
 
 // UpdateTRBRequest is the resolver for the updateTRBRequest field.
 func (r *mutationResolver) UpdateTRBRequest(ctx context.Context, id uuid.UUID, changes map[string]interface{}) (*models.TRBRequest, error) {
-	return resolvers.TRBRequestUpdate(ctx, id, changes, r.store)
+	return resolvers.UpdateTRBRequest(ctx, id, changes, r.store)
+}
+
+// CreateTRBRequestAttendee is the resolver for the createTRBRequestAttendee field.
+func (r *mutationResolver) CreateTRBRequestAttendee(ctx context.Context, input model.CreateTRBRequestAttendeeInput) (*models.TRBRequestAttendee, error) {
+	return resolvers.CreateTRBRequestAttendee(ctx, r.store, &models.TRBRequestAttendee{
+		TRBRequestID: input.TrbRequestID,
+		EUAUserID:    input.EuaUserID,
+		Component:    input.Component,
+		Role:         models.PersonRole(input.Role),
```

Comment:
if GQL is changed to use `PersonRole`,  you would no longer need to cast here.

Reply:
Whoops, didn't notice these spots